core/math/color.cpp

```diff
@@ -432,6 +432,26 @@ Color Color::get_named_color(int p_idx) {
 	return named_colors[p_idx].color;
 }
 
+String Color::to_named() const {
+	NamedColor closest_named_color = {};
+	float closest_distance_squared = std::numeric_limits<float>::infinity();
```

Comment:
Use Godot's own constant for consistency with the rest of the codebase:

```suggestion
	float closest_distance_squared = Math::INF;
```

Reply:
I accept. It might be a good idea to cast to `(float)` since that sounds like a narrowing conversion, but I'll leave it up to you.